src/components/ManageUsers/manage-users.css

```diff
@@ -0,0 +1,84 @@
+.usersearch__h1 {
+  font-size: 16px;
+  color: #e3893e;
```

Comment:
```suggestion
  color: var(--color-primary);
```

Change everywhere where you are using our orange color

Reply:
Has done for manage-users.css